scripts/deploy-v2.js

```diff
@@ -210,7 +221,8 @@ async function main() {
 
 
     console.log(`Deploying Network_V2 With Registry...`);
-    const registryReceipt = await Deployer(NetworkRegistry, [networkToken.contractAddress, 1000, ownerAddress, 10000]);
+    const treasuryAddress = argv.treasuryAddress ? argv.treasuryAddress: ownerAddress;
+    const registryReceipt = await Deployer(NetworkRegistry, [networkToken.contractAddress, 1000, treasuryAddress, 10000]);
```

Comment:
missing arguments for "network creation value", because I'm thinking we dont want that to be `1000`

Reply:
On production, we can change adjust that later on the Network registry.